packages/react-badge/Spec.md

```diff
@@ -34,7 +34,7 @@ type BadgeAppearance = 'filled' | 'outline' | 'ghost' | 'tint';
 
 type BadgeShape = 'rounded' | 'square' | 'circular';
 
-type BadgeSize = 'smallest' | 'smaller' | 'small' | 'medium' | 'large' | 'larger' | 'largest';
+type BadgeSize = 'tiny' | 'extra-small' | 'small' | 'medium' | 'large' | 'extra-large';
```

Comment:
I know the figma talked about smallest, smaller.  Levi and I had a long discussion about sizing issues and using relative size words makes it sound like it is a scale against a baseline size.  We opted for the common semantic size names (a la bootstrap from tiny up to jumbo).  Because the larger and largest sizes are the same, it didn't make sense to keep the largest value as jumbo in this case.